nibiru/clients/query/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .dex import DexQueryClient  # noqa
+from .perp import PerpQueryClient  # noqa
+from .pricefeed import PricefeedQueryClient  # noqa
+from .vpool import VpoolQueryClient  # noqa
```

Comment:
I'd like to stick to this structure for imports: [Google Python Style Guide - 2.2.4](https://google.github.io/styleguide/pyguide.html#224-decision)

> Do not use relative names in imports. Even if the module is in the same package, use the full package name. This helps prevent unintentionally importing a package twice.

- Use import `x` for importing packages and modules.
- Use from `x` import `y` where `x` is the package prefix and `y` is the module name with no prefix.
- Use from `x import y as z` if two modules named `y` are to be imported, if `y` conflicts with a top-level name defined in the current module, or if `y` is an inconveniently long name.
- Use `import y as z` only when `z` is a standard abbreviation (e.g., `np` for `numpy`).